.github/workflows/test.yml

```diff
@@ -21,7 +21,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        go-version: [1.17.x, 1.19.x, 1.20.x]
+        go-version: [1.17.x, 1.20.x, 1.21.x]
```

Comment:
Do we still need 1.17 ?

Reply:
I'm not sure. The original plan was to maintain support for older releases in a stable branch, but since this branch ended up a very long lived one, we had to drop 1.16 support in runc 1.1.6 because of #3817.

Let it stay for now (I hope we'll release 1.2.0 soon and more-or-less forget about 1.1.x)